src/Template/Element/random_sentence_header.ctp

```diff
@@ -25,40 +25,47 @@
  * @link     https://tatoeba.org
  */
 
-$this->Html->script(JS_PATH . 'sentences.show_another.js', array('block' => 'scriptBottom'));
```

Comment:
You should remove the file `webroot/js/sentences.show_another.js` too, since it’s not used anywhere else.